src/main/java/pl/connectis/Spotify_API_CLI/authorization/AuthorizationController.java

```diff
@@ -0,0 +1,37 @@
+package pl.connectis.Spotify_API_CLI.authorization;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.concurrent.locks.Condition;
+import java.util.concurrent.locks.Lock;
+
+@Slf4j
+@RestController
+public class AuthorizationController {
```

Comment:
nie dokońca rozumiem jak to działa. Kto woła endpoint `/collect`? OAuth2 Flow sam wie ze musi zawołać `/collect`?

Reply:
W zasadzie tak. W ustawieniach Spotify dla utworzonego klienta podałem redirect uri na `/collect`. Po prostu w ten sposób zautomatyzowałem częściowo proces autoryzacji przy flow authorization code.